en/ExternalFiles.md

```diff
@@ -16,6 +16,32 @@ These are by default set to values that probably make sense for your operating s
 
 To change the external viewer settings, go to **Options → Preferences → External programs**.
 
+## Pushing to external viewer application
+
+JabRef allows you to push any entries in your main window to an external viewer through the push-to-external application feature. You would need to first select the entries in your entry table that you would like to export before using the feature. Once you have done so, go to the tools submenu and click on the push-to-external application button to the left of the **Generate BibTeX keys** button. By default the external viewer used to push exports is TeXstudio.
+
+On MacOS:
+![Push to External MacOS](images/Push-External-Button-MacOS.png)
+
+On Windows:
+![Push to External Windows](images/Push-External-Button-Windows.png)
+
+JabRef also allows you to change the external viewer application you would like to push your exports to. To do so, first go to **Options → Preferences → External programs**. Under the **Push applications** section click on the **Application to push entries to** field. This will cause a dropdown menu to appear, from which you are able to select from a list of the external viewers you have configured.
+
+![Select External Application](images/During-Application-Selection.png)
+
+Once you have made your selection and click **Save**, the push-to-external application button icon will change to match that of the selected external viewer application.
+
+![New Application After Select](images/After-Application-Selection.png)
+
+When you click on the push-to-external application button, JabRef will export your selected entries to a file generated by the selected external viewer application. As an example, here is what happens when you export one entry to TexStudio.
+
+![Initial Push to External Export](images/Initial-Push-Export.png)
```

Comment:
Thank you for installing the tools. The thing is, that one should have a `.tex` file opened. You can use that file: https://github.com/latextemplates/scientific-thesis-template/blob/master/latexhints-english.tex#L43

Adding a `cite` command to a BibTeX file leads to compilation errors.

![grafik](https://user-images.githubusercontent.com/1366654/68100093-55ebb080-fec6-11e9-8e72-b8184df8f065.png)

Reply:
I don't quite understand what you want. When I export a bibliography in JabRef using push-to-external application, it gets all existing entries in the bibliography and I get the following: 

After exporting first entry to TexStudio
<img width="1280" alt="Screen Shot 2019-11-06 at 5 11 21 PM" src="https://user-images.githubusercontent.com/33012425/68342396-7e3b0100-00b8-11ea-99d0-f543d76de610.png">

After exporting additional (second) entry to TexStudio
<img width="1280" alt="Screen Shot 2019-11-06 at 5 14 25 PM" src="https://user-images.githubusercontent.com/33012425/68342576-e7bb0f80-00b8-11ea-8bc1-cb363e6bf77a.png">

Should I use these screenshots instead? Or are you saying that the exported file has to be in `.tex` format (which I then show screenshots for)? If it's the latter, I'd appreciate it if you could guide me on how to get the proper `.tex` file for the exported entries via JabRef (or am I supposed to use the file you linked as an example). Please let me know.